forum/models/users.py

```diff
@@ -0,0 +1,133 @@
+"""Users Class for mongo backend."""
+
+from forum.models.abstract import AbstractModel
+
+
+class Users(AbstractModel):
+    """
+    Users class for cs_comments_service user model
+    """
+
+    def __init__(self, collection_name="users", client=None):
+        """
+        Initializes the Users class.
+
+        Args:
+            collection_name: The name of the MongoDB collection.
+            client: The MongoDB client.
+
+        """
+        super().__init__(collection_name, client)
+
+    def get(self, external_id):
```

Comment:
Similarly, we can update this to:

self.get_collection.find_one(kwargs), there might be case where we want to get a user based on some other parameter.

Reply:
I think we should  keep it to only `id` since we only need to fetch one user at a time and id is unique.

Reply:
Done